examples/existing-module/main.tf

```diff
@@ -1,7 +1,8 @@
 # Provision Intel Optimized AWS MySQL server
 
 module "aws-mysql-parameter-group" {
-  source = "intel/aws-mysql-parameter/intel"
+  #source = "intel/aws-mysql-parameter/intel"
+  source = "../../"
```

Comment:
Remove the
  source = "../../" 
and uncomment the 
  source = "intel/aws-mysql-parameter/intel"